CSS/style.css

```diff
@@ -0,0 +1,108 @@
+*{
+    box-sizing: border-box;
+}
+
+/* Navigation */
+
+#header{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: whitesmoke;
+}
+#header-img{
+    
+}
+
+#header #nav-bar{
+    text-decoration: none;
+}
+
+#header #nav-bar li{
+    /* display: flex; */
+    /* align-items: right; */
+    /* float: right; */
+    display: inline-flex;
+    list-style: none;
+    justify-content: space-around;
+    padding-right: 50px;
+    font-size: 30px;
+    
+}
+#header #nav-bar a{
+    /* justify-content: space-around; */
+    text-decoration: none;
+}
+
+/* form */
+
+#form{
+    /* align-items: center; */
+    height: 300px;
+   
+    /* text-align: center; */
+    padding-top: 60px;
+    background-color: whitesmoke;
+    
+}
+
+/* Video section */
+
+#video{
+    height: 600px;
+    
```

Comment:
I think it would be looking nice if you remove unnecessary empty lines in your code. Remove them to make your code clean

Reply:
- [ ] Please make your comment more direct and kindly always as for changes in a friendly way for example you can do it like that:
`Kindly remove all unnecessary empty lines inside your code to make your code cleaner and readable`

Reply:
- [ ] Please remove unnecessary empty lines to make your code readable.